### PR TITLE
feat(console): log details page

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Implement LogDetails page.

## Linear Issue Reference

LOG-2605

## Testing

截屏2022-06-07 下午2 25 22

### CODE REVIEW

File: packages/console/src/pages/AuditLogDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                  <div className={styles.label}>{t('log_details.event_type')}</div>
+                  <div>{data.type}</div>
+                </div>
+                <div className={styles.infoItem}>
+                  <div className={styles.label}>{t('log_details.application')}</div>
+                  <div>
+                    {data.payload.applicationId ? (
+                      <ApplicationName isLink applicationId={data.payload.applicationId} />
+                    ) : (
+                      '-'
```
Comment (xiaoyijun) [inline packages/console/src/pages/AuditLogDetails/index.tsx:57]: nit: It seems that the `<ApplicationName />` component will handle the `-` case.

File: packages/console/src/pages/AuditLogDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                  </div>
+                </div>
+                <div className={styles.infoItem}>
+                  <div className={styles.label}>{t('log_details.ip_address')}</div>
+                  <div>{data.payload.ip ?? '-'}</div>
+                </div>
+                <div className={styles.infoItem}>
+                  <div className={styles.label}>{t('log_details.user')}</div>
+                  <div>
+                    {data.payload.userId ? <UserName isLink userId={data.payload.userId} /> : '-'}
```
Comment (xiaoyijun) [inline packages/console/src/pages/AuditLogDetails/index.tsx:68]: ditto.

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

**Comment by xiaoyijun** [comment]
If I jump to the user detail page from this log details page, I can not nav back by the back button of the browser.

**Comment by wangsijie** [comment]
@xiaoyijun It will be opened in a new tab.

**Comment by xiaoyijun** [comment]
> @xiaoyijun It will be opened in a new tab.

Good!

File: packages/console/src/pages/AuditLogDetails/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/AuditLogDetails/index.tsx:57]: Maybe we could support it and let the `<XXXname>` component handle all the case?

**Pull request merged**